.circleci/config.yml

```diff
@@ -1,41 +1,134 @@
-version: 2
+version: 2.1
+defaults: &defaults
+  working_directory: ~/user-data-for-fraud-prevention
+  docker:
+    - image: circleci/node:10.15.3-browsers
+  environment:
+    TZ: "/usr/share/zoneinfo/America/Los_Angeles"
```

Comment:
Do we need to set this? Is this mandatory?

Reply:
Nope! I can remove it if you want.

Reply:
Cool.. Please remove it for now.